pages_/turn/[turnId].tsx

```diff
@@ -130,6 +130,7 @@ const MyTurn = ({
             {t('common:my-turn')}
           </Button>
 
+          {/* TODO NICO: Add "myturn__number--disabled" class for disabled myturn__number */}
```

Comment:
is this comment still valid?

Reply:
@ramirotw Yes. For "expired" or already completed Turns we need to add that class to the number to change its style. It is not ok to keep it as a blue number. We need to also add some wording stating that "Este turno está completado" or something like that.

Reply:
@ramirotw Right now, the only indication of Turn status is in the Home turn list. We also need some visual indication of turn-expiration in the My Turn screen as well.